pybossa/api/task.py

```diff
@@ -87,3 +98,14 @@ def _sign_item(self, item):
         if current_user.admin or \
            current_user.id in get_project_data(project_id)['owners_ids']:
             sign_task(item)
+
+    def _select_attributes(self, data):
+        project_id = data['project_id']
+        if current_user.admin or \
+            (current_user.subadmin and
+                current_user.id in get_project_data(project_id)['owners_ids']):
+            return data
+        else:
+            tmp = copy.deepcopy(data)
```

Comment:
why `deepcopy`?

Reply:
so as to not update the actual object and pass to regular user an object w/o gold_ans in it